lib/routes/minecraft/news.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cheerio from 'cheerio';
+
+export const route: Route = {
+    path: '/news',
+    categories: ['game'],
+    example: '/minecraft/news',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['minecraft.net/'],
+        },
+    ],
+    name: 'Minecraft News',
```

Comment:
No need to include namespace name https://github.com/DIYgod/RSSHub/blob/5a188a3c5c949dc66802529fcad0291d7ef34550/lib/routes/minecraft/namespace.ts#L4 again in route name.